src/App.js

```diff
@@ -1,15 +1,30 @@
 import React from "react";
 
+const todoList = [
+  {
+    id: 1,
+    title: "code",
+  },
+  {
+    id: 2,
+    title: "go to store",
+  },
+  {
+    id: 3,
+    title: "attend group session",
+  },
+];
+
 function App() {
   return (
     <div style={{ textAlign: "center" }}>
       <header>
```

Comment:
there's still this `header` element left over from the originally-generated code, as well as the `style` tag inside the `div`. 

This is causing your page to render a little strangely, but everything will work great after you remove those. 

<img width="1792" alt="Screen Shot 2023-06-06 at 9 31 38 AM" src="https://github.com/evaw277/react-todo/assets/12771651/7a782b7b-56d8-4443-9a74-82484362ec64">